src/core_plugins/timelion/server/routes/run.js

```diff
@@ -4,7 +4,10 @@ import chainRunnerFn from '../handlers/chain_runner.js';
 const timelionDefaults = require('../lib/get_namespaced_settings')();
 
 function replyWithError(e, reply) {
-  reply({ title: e.toString(), message: e.toString(), stack: e.stack }).code(400);
+  reply({
```

Comment:
I think this should throw a boom.badImplementation instead (500) instead.  If we want the stack trace still I think it makes sense to log it with the server logger.

The response still has some info, and it can be a little strange to understand:
```
{
    "title": "TypeError: Cannot read property 'to' of undefined",
    "message": "TypeError: Cannot read property 'to' of undefined"
}
```